modules/031-local-path-provisioner/docs/USAGE.md

```diff
@@ -14,3 +14,10 @@ spec:
   - system
   path: "/opt/local-path-provisioner"
 ```
+
+Notes:
+
+- This example will create `localpath-system` storage class which **must** be used by pods for everything to work
+- Volumes created by provisioner will have delete retention policy which is hardcoded ([issue](https://github.com/deckhouse/deckhouse/issues/360))
+- If provisioner will be delete before claims folders will not be deleted from node
+- Note that in example `system` node is used which probably will have some taints so pods **must** have corresponding tolerations
```

Comment:
this isn't needed due to prometheus has tolerations for system nodes by default.